configuration/rules-dsl.md

```diff
@@ -301,6 +300,30 @@ then
 end
 ```
 
+{: #transformations}
+### Transformations
+
+openHAB [Transformation services]({{base}}/addons/transformations.html) can also be used in rules to **transform/translate/convert data**.
```

Comment:
Consider "openHAB Transformation services may also be used..."  Why 'may' and not 'can'?  Hum - its a native English speaker thing.  Can means it has the capability to do what is mentioned - it is possible.  May means it might do what is mentioned, but it might not do it either.  In this case, a user may choose to use them in rules to transform/translate/convert data, but they do not HAVE to do it.

Again, in this sentence, I do not understand the significance of the bold text and would eliminate it, but I do not have a strong feeling about this.

Reply:
Thanks. "may" is indeed one of those words I do not use often... I may try to remember that! 😜